client/src/main/scala/skuber/Volume.scala

```diff
@@ -17,6 +17,16 @@ object Volume {
       directory: Option[String] = None)
     extends Source
 
+  sealed trait ProjectedSource
+  case class Projected(defaultMode:Option[Int] = None,
+                       sources: List[ProjectedSource]) extends Source
+  case class ProjectedSecret(name: String,
+                             items: Option[List[KeyToPath]] = None) extends ProjectedSource
+  case class ProjectedConfigMap(name: String,
+                                items: Option[List[KeyToPath]] = None,
+                                mode: Option[Int] = None) extends ProjectedSource
```

Comment:
The Kubernetes API reference (https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#volumeprojection-v1-core) suggests there is no field called `mode` here, but there is a boolean `optional` field.

Reply:
fixed